src/manifolds/PowerManifold.jl

```diff
@@ -338,28 +322,45 @@ function get_coordinates(
     M::AbstractPowerManifold,
     p,
     X,
-    B::PrecomputedPowerOrthonormalBasis,
-)
+    B::CachedBasis{<:AbstractBasis{ℝ},<:PowerBasisData,ℝ}
+) where {𝔽}
```

Comment:
I can't see `𝔽` in types of arguments.

Reply:
Oh that's just a leftover from a lot of trying.